### PR TITLE
Nc news 6

### PR DESCRIPTION
**Pull request by nmb882003**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -33,8 +35,10 @@ app.use((err, req, res, next) => {
     res.status(500).send({ msg: 'Internal server error' });
 });
 
+/*
 app.listen(8080, () => {
     console.log(`Listening on port 8080`);
 });
+*/
```
Comment (nmb882003) [inline app.js:42]: So... yesterday I learned that if I don't comment this out when running jest/Supertest then it gradually breaks my internet connection. Who knew??

Comment (nmb882003) [inline app.js:42]: So... yesterday I learned that if I don't comment this out when running Supertest then it gradually breaks my internet connection. Who knew??

Comment (DCrawley94) [reply app.js:42]: Oh no! We don't need it unless we're using something like Insomnia to be fair so don't worry about it :)

**Review by DCrawley94 — APPROVED** [review]
Looking good!

**Pull request merged**